### DISCUSSION TITLE
Homeassistant: who can adopt this as addon?

### DISCUSSION THREAD

**Discussion by senna1992** [Stirling-Tools/Stirling-PDF#1977] [Polls]
Hi there, anyone interested in making this available for Homeassistant?

I would love to see this, but without prior experience to stuff alike I can't do it myself.
Looking for someone to work together on this!

**Comment by pablodelarco** [comment]
Hi! @senna1992
I created a Stirling-PDF add-on that works in Home Assistant :smiley:
I'm not an expert but it's working. 
Please, feel free to improve it or change anything you consider.

This is the repo: [https://github.com/pablodelarco/stirling_pdf_addon](https://github.com/pablodelarco/stirling_pdf_addon)

**Reply by Frooodle** [reply]
Might be best to use the docker or docker fat file instead of the lite version to get full functionality 

Also would love to fork this and add it under the official Stirling pdf if you fancy changing your licence!

**Reply by senna1992** [reply]
@pablodelarco thanks for mentioning!

i am no expert when it comes to creating staple and sustainable addons. 
@Frooodle i'd love to have an official addon, so if you want i can test early version :)

**Reply by pablodelarco** [reply]
@Frooodle I’ve updated the add-on with the fat version and added the MIT license, so you can fork it. Looking forward to seeing the improvements you implement! 😊
